### PR TITLE
Add some type operators

### PR DESCRIPTION
**Pull request by Martoon-00**

### CODE REVIEW

File: universum.cabal [hunk 1]
```diff
@@ -71,10 +72,11 @@ library
     safe                  >= 0.3    && <0.4,
     stm                   >= 2.4    && <2.5,
     text                  >= 1.2    && <1.3,
-    utf8-string           >= 1.0    && <1.1,
     text-format,
     transformers          >= 0.4    && <0.6,
+    type-operators        == 0.1.0.4,
```
Comment (neongreen) [inline universum.cabal]: Why such a strict constraint?

File: universum.cabal [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name:                universum
-version:             0.2.1
+version:             0.2.2
```
Comment (neongreen) [inline universum.cabal]: Update the changelog as well

File: universum.cabal (hunk 1 above)
Comment (Martoon-00) [inline universum.cabal]: In earlier versions they had no `<+>` and `<=>` operators.
On the other side, after `0.1.0.0` they removed `%` operator. Looks like author doesn't care much about backward compatibility

File: universum.cabal (hunk 2 above)
Comment (Martoon-00) [inline universum.cabal]: Roger!

File: src/TypeOps.hs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+#if __GLASGOW_HASKELL__ <= 710
+import           GHC.Prim              (Constraint)
+#else
+import           Data.Kind             (Constraint)
+#endif
+
+import           Control.Type.Operator (type ($), type (<+>))
+
+-- | Map several constraints over a several variables.
```
Comment (Martoon-00) [inline src/TypeOps.hs]: Kek, typo

File: universum.cabal (hunk 1 above)
Comment (neongreen) [inline universum.cabal]: mda

**Pull request merged**